### DISCUSSION TITLE
Artillery Pro Subscriptions

### DISCUSSION THREAD

**Discussion by jessehattabaugh** [artilleryio/artillery#1451] [Artillery]
Are these actually available? I haven't gotten a response from the email form. Are there any active subscribers currently? My company is evaluating Artillery versus K6 and we need a cloud-scale solution. Why does the Free tier require a license, and why does it take so long to get a response?

**Comment by hassy** [comment]
hi @jessehattabaugh 👋 thanks for the nudge & apologies for the long wait. We're working through a large backlog of requests for Dev accounts. We'll set one up for you right away.

Artillery Pro runs fully on-prem so all accounts require a license right now.
